packages/client/src/components/CardMessange/index.tsx

```diff
@@ -1,30 +1,125 @@
+import React from 'react'
 import * as RB from '@mui/material'
-
+import ImageIcon from '@mui/icons-material/Image'
+import FavoriteIcon from '@mui/icons-material/Favorite'
+import { useAppDispatch, useAppSelector } from '../../store/index'
+import { selectForumData } from '../../store/slices/ForumSlice'
+import { selectUserData } from '../../store/slices/UserSlice'
+import FormMessange from '../FormSendMess'
+import { Comment } from '../../store/types'
+import { createLike } from '../../store/actions/ForumActionCreators'
 interface Props {
-  name: string
-  text: string
-  data: string
+  like: boolean | undefined
+  comment: Comment
+  childComment: Comment[]
 }
 
 const CardMessange = (props: Props) => {
+  const dispatch = useAppDispatch()
+  const { status } = useAppSelector(selectForumData)
+  const { userData } = useAppSelector(selectUserData)
+  const [open, setOpen] = React.useState(false)
+  const handleOpen = () => {
+    setOpen(!open)
```

Comment:
Для вычисления состояния на основе предыдущего надежнее [использовать коллбек](https://beta.reactjs.org/reference/react/useState#setstate)

```suggestion
    setOpen(prevOpen => !prevOpen)
```